packages/dso-toolkit/src/styles/components/_button.scss

```diff
@@ -135,6 +135,21 @@ dso-label[status="danger"],
   }
 }
 
+dso-banner {
```

Comment:
Is het niet gebruikelijk dat we deze styling dan ook op de selector voor het CSS component plaatsen en weghalen/niet meenemen naar de stylesheet in `@dso-toolkit/styling` ?

Reply:
Zeker, goed punt. Het is tenslotte `button` styling in een context